mongodbatlas/resource_mongodbatlas_search_index.go

```diff
@@ -48,9 +47,10 @@ func returnSearchIndexSchema() map[string]*schema.Schema {
 			Required: true,
 		},
 		"analyzers": {
-			Type:     schema.TypeList,
+			Type:     schema.TypeString,
```

Comment:
making sure I get this right, analyzers and field mapping for terradform are now "strings" of json that we just map to the go client `interface{}` type correct? general question nothing blcoking here